src/data_capture/video_capture/launch/capture.launch

```diff
@@ -2,6 +2,7 @@
 
     <arg name="image_topic" default="camera/color/image_raw"/>
     <arg name="is_record_topic" default="video_capture/is_record"/>
+    <arg name="is_memory_usage_exceeded_topic" default="/is_memory_usage_exceeded"/>
```

Comment:
I don't think that this topic should be global by default (leading `/`). If we have multiple computers, like QT does, we might want them to be local. 

In ROS, it is generally bad practice to use the global namespace.